puppet/manifests/overcloud_compute.pp

```diff
@@ -288,6 +287,21 @@
     class { 'onos::ovs_computer':
       manager_ip => $controller_ips[0]
     }
+  } elsif 'vpp' in hiera('neutron::plugins::ml2::mechanism_drivers') {
+    $tenant_nic = hiera('tenant_nic')
+    $dpdk_tenant_port = hiera("${tenant_nic}", false)
+    if ! $dpdk_tenant_port { fail("Cannot find physical port name for logical port ${dpdk_tenant_port}")}
+
+    $tenant_nic_vpp_str = inline_template("<%= scope.function_hiera(['${dpdk_tenant_port}_vpp_str']) -%>")
```

Comment:
can this be:

     $tenant_nic_vpp_str = hiera("${dpdk_tenant_port}_vpp_str", false)

instead? I think it's a bit simpler